layouts/shortcodes/ecosystem/registry/search-form.html

```diff
@@ -34,10 +34,54 @@
 {{ end -}}
 {{ $types = $types | uniq | sort -}}
 
+<!--Custom css for the inline help-->
```

Comment:
Please move custom CSS into this file:

https://github.com/open-telemetry/opentelemetry.io/blob/main/assets/scss/_registry.scss

Doing so, make sure that the CSS is only applied on that page, e.g. right now if we use "btn-outline-success" class on any other page the hover::after would also be applied

Reply:
Thanks mentor I will update it.

Reply:
Please make sure that you provide the update today, such that I still can review it. I will not be available for more feedback afterwards until the end of the application phase